docs/community/team.md

```diff
@@ -0,0 +1,17 @@
+# GradleUp Team
+
+The project is led by a small team of passionate Gradle users and contributors.
+The team is decentralized and decisions are made based on consensus and common sense.
+
+## Members
+
+- [Alex Nordlund (@deepy)](https://github.com/deepy)
+- [Jean-Michel Fayard (@jmfayard)](https://github.com/jmfayard)
+- [Louis CAD (@LouisCAD)](https://github.com/LouisCAD)
+- [Martin Bonnin (@martinbonnin)](https://github.com/martinbonnin)
+- [Pablisco (@pablisco)](https://github.com/pablisco)
+- [Said Tahsin Dane (@tasomaniac)](https://github.com/tasomaniac)
```

Comment:
We're missing you here :)

Reply:
<3

Reply:
I will add the Shadow folks in a separate patch

```suggestion
- [Said Tahsin Dane (@tasomaniac)](https://github.com/tasomaniac)
- [Nicola Corti](https://github.com/cortinico)
```